packages/node-feature-discovery/v0.16.2+1/package.yaml

```diff
@@ -0,0 +1,16 @@
+# yaml-language-server: $schema=https://glasskube.dev/schemas/v1/package-manifest.json
+
+name: "node-feature-discovery"
+shortDescription: "Detects hardware features available on each node in a Kubernetes cluster,
+  and advertises those features using node labels."
+iconUrl: "https://avatars.githubusercontent.com/u/1728152"
```

Comment:
NFD is a project owned by k8s-sigs, not Nvidia. I think the icon should be the "normal" k8s icon we use for some other packages already.